templates/repo/header.tmpl

```diff
@@ -11,6 +11,7 @@
 				<a href="{{AppSubUrl}}/{{.Owner.Name}}">{{.Owner.Name}}</a>
 				<div class="divider"> / </div>
 				<a href="{{$.RepoLink}}">{{.Name}}</a>
+				{{if and .RelAvatarLink .IsPrivate}}<i class="text gold octicon octicon-lock"></i>{{end}}
```

Comment:
Two things I'm wondering...

1. Should it *always* be on the right side, rather than only if there's an avatar? Then it would be consistent.
a. Alternatively, should the icons just always show up on the left, next to the avatar if it exists?

2. An icon should show regardless of whether it's private or not, similar to how it would look with no avatar. At least in my opinion.
```suggestion
				{{if not .RelAvatarLink}}<i class="text gold octicon octicon-{{if .IsPrivate}}lock{{else if .IsMirror}}repo-clone{{else if .IsFork}}repo-forked{{else}}repo{{end}}"></i>{{end}}
```

Reply:
#### My opinion
* icon position:
  * I would place the small ones always on **right** side. So text will not jump around if there is something special on repository (special: private,mirror,fork,archived)
* I would only show the small icons, **when** it is hidden by **avatar**.
  When no avatar is set then there would be a second icon with the same information
* furthermore i would't show a symbol on right side if it is a normal repository
   * so you see with a short glance if it is a normal repo or not (i know, with no avatar the normal repo icon is shown, but the size is bigger.)
* you're right i missed the other special icons, but the first condition in your proposal isn't correct. Your proposal would only be rendered if avatar is not set.
-> I will add the other icons there

Reply:
Ah, good catch on my suggestion.